hacep-core/src/main/java/it/redhat/hacep/HACEPImpl.java

```diff
@@ -92,6 +88,10 @@ public void start() {
                 this.rulesManager.start(groupId, artifactId, version);
                 infoCache.addListener(new UpdateVersionListener(this.router, this.rulesManager));
 
+                infoCache.put(Router.SUSPEND, "0");
+                infoCache.put(Router.RESUME, "0");
```

Comment:
I'd use putIfAbsent otherwise a restarting node will trigger SUSPEDNA and then RESUME in all the other nodes